MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -73,7 +73,7 @@ open class BaseInstructionsBannerView: UIControl {
     
     func set(_ instruction: VisualInstruction?) {
         let secondaryInstruction = instruction?.secondaryTextComponents
-        primaryLabel.numberOfLines = secondaryInstruction == nil ? 2 : 1
+        primaryLabel.numberOfLines = 1
```

Comment:
I'm sort of digging this change. This means we will abbreviate more, but will only use one line ever per primary/secondary instruction.

Reply:
It would simplify the layout and snapshot tests. Note that this method https://github.com/mapbox/mapbox-navigation-ios/blob/2890d2f649b5d4f62285747d601c94cf565a6aad/MapboxNavigation/InstructionsBannerViewLayout.swift#L113-L117
and accompanying arrays of constraints would be obsolete if we decide to make this change consistent.